src/Moryx.ControlSystem/Simulation/ISimulationDriver.cs

```diff
@@ -7,7 +7,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Moryx.Simulation
```

Comment:
Would you be so kind as to put an entry into the migration guide for this package change? 😬